OrangeDesignSystem/Sources/OrangeDesignSystem/Modules/About/Internal/LitsItems/AppNews/AppNewsViews.swift

```diff
@@ -34,31 +34,59 @@ struct AppNewsList: View {
 
     var body: some View {
         ScrollView {
-            if releaseDescriptions.isEmpty {
-                Text(°°"modules.about.app_news.no_news")
-            } else {
-                ForEach(releaseDescriptions, id: \.version) { releaseDescription in
-                    AboutReleaaseDescriptionEntry(releaseDescription: releaseDescription)
-                        .padding(.horizontal, ODSSpacing.m)
-                    Divider()
-                }
+            switch viewModel.releaseDescriptions {
+            case .loading:
+                loadingView()
+            case let .loaded(releaseDescription) where !releaseDescription.isEmpty:
```

Comment:
and dedicated case for empty

Reply:
This case is managed in the `default` case ; we don't have more screens specified yet so I suppose if we don't have any values the file is not well filled and we face an error.